chrome-launcher/chrome-launcher.ts

```diff
@@ -13,7 +13,7 @@ import {DEFAULT_FLAGS} from './flags';
 import {makeTmpDir, defaults, delay} from './utils';
 import * as net from 'net';
 const rimraf = require('rimraf');
-const log = require('lighthouse/lighthouse-core/lib/log');
+const log = require('lighthouse-logger');
```

Comment:
oh boy haha, can we just grab the lighthouse org instead? 😃 
* `require('@lighthouse/logger')`
* `require('@lighthouse/chrome-launcher')`
* `require('@lighthouse/all-the-things')`